frontend/src/scenes/insights/Insights.tsx

```diff
@@ -56,6 +56,51 @@ function InsightHotkey({ hotkey }: { hotkey: HotKeys }): JSX.Element {
     return !isMobile() ? <span className="hotkey">{hotkey}</span> : <></>
 }
 
+function ComputationTimeAndRefresh({
```

Comment:
Nice refactor! Would it be possible to move this into a separate component file? I feel like this would be a super reusable component.